portlets/src/main/webapp/vue-app/rule/components/RuleList.vue

```diff
@@ -441,7 +441,7 @@ export default {
     collapseButtonn(rule) {
       this.editedrule = rule;
       this.isShown = !this.isShown;
-      this.editedrule.description =  this.$t(`exoplatform.gamification.gamificationinformation.rule.description.${this.editedrule.title}`,this.editedrule.description) ;
+      this.editedrule.description =  (this.$t(`exoplatform.gamification.gamificationinformation.rule.description.${this.editedrule.title}`),this.editedrule.description );
```

Comment:
please use this instead :
https://github.com/Meeds-io/gamification/blob/1cf5787acc387de11bb821aaedec75f3f9756df8/portlets/src/main/webapp/vue-app/rule/components/RuleList.vue#L458